src/assets/stylesheets/components/_button.scss

```
@@ -0,0 +1,11 @@
+.sign-in-btn {
+  width: 100%;
+  height: 56px;
```

Change to use rem function 🙏 
```suggestion
  height: rem(56px);
```

---

Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/6f7d8911e5efec5fc7ac24ea69f8d8a49e9dda7d#diff-742a38b49460f2989e2629c43b08487197067f1991bdfc656867202ee4c57911